cmd/fetch/fetch.go

```diff
@@ -56,6 +59,33 @@ func NewCmdFetch() *cobra.Command {
 	return fetchCmd
 }
 
+func diagToEventType(d diag.Diagnostic) string {
+	if strings.HasSuffix(d.Error(), "i/o timeout") {
+		return "io_timeout"
+	}
+	return ""
+}
```

Comment:
I can see this growing into many event types, we currently have fetch and policy, but in this case we can have more.
Maybe call this event type "diag" and give it sub types and some meta information as occurrence count etc' and by no means to pass the "error" itself as that can have user information we don't want.

ill let @bbernays  weigh in some more here.